cvat/apps/engine/tests/test_rest_api.py

```diff
@@ -3086,62 +3017,22 @@ def test_api_v2_tasks_id_export_no_auth(self):
         self._run_api_v2_tasks_id_export_import(None)
 
 def generate_image_file(filename):
-    f = BytesIO()
     gen = random.SystemRandom()
     width = gen.randint(100, 800)
     height = gen.randint(100, 800)
-    image = Image.new('RGB', size=(width, height))
-    image.save(f, 'jpeg')
-    f.name = filename
-    f.seek(0)
-
+    f = _generate_image_file(filename, size=(width, height))
```

Comment:
In general, it's not a good idea to have randomness in tests, because it makes the tests harder to reproduce. In this case, I decided to factor out only the controlled-size version, because this random variant is used everywhere in this file. The extracted function can be used in other tests for other components, there are lots of duplicated code to remove.

Reply:
Okay, that's fair enough, but I still object to the fact that both of these functions are named `generate_image_file`. Can you rename the randomized one so that it's clear it's a different function?